format/src/main/scala/org/scalafmt/internal/TreeDocOps.scala

```diff
@@ -165,6 +165,10 @@ object TreeDocOps {
     SimpleExpr.wrap(lhs) + sep + rhs
   }
 
+  def dPath2(lhs: Tree, sep: Doc, rhs: Doc): Doc = {
```

Comment:
Can we come up with a more descriptive name here? Is this method necessary? Seems we might as well inline it.